extensions/opentelemetry/deployment/src/test/java/io/quarkus/opentelemetry/deployment/OpenTelemetryContinuousTestingTest.java

```diff
@@ -22,8 +22,22 @@ public class OpenTelemetryContinuousTestingTest {
                     .addClasses(TestSpanExporter.class, TestSpanExporterProvider.class)
                     .addAsResource(new StringAsset(TestSpanExporterProvider.class.getCanonicalName()),
                             "META-INF/services/io.opentelemetry.sdk.autoconfigure.spi.traces.ConfigurableSpanExporterProvider")
+                    //                    .add(new StringAsset(
+                    //                                    "quarkus.otel.traces.exporter=test-span-exporter\n" +
+                    //                                            "quarkus.otel.devservices.enabled=false\n"),
+                    //                            "application.properties"))
+                    //            .setTestArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                    //                    .addClass(TracerRouterUT.class));
+                    //    @RegisterExtension
```

Comment:
Sorry, this should not have been part of the commit. I was trying to prevent some last tests from running dev service. I have asked @alesj about help with those tests.